S4TF_Tutorial_3.ipynb

```diff
@@ -0,0 +1,1768 @@
+            "text/plain": [
+              "<Figure size 288x288 with 2 Axes>"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          }
+        },
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "None\n"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 10
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "liZMNGVSCLgd",
+        "colab_type": "text"
+      },
+      "source": [
+        "Display the first 25 images from the *training set* and display the class name below each image. Verify that the data is in the correct format and we're ready to build and train the network."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "RHBungPfCLrg",
+        "colab_type": "code",
+        "outputId": "28c588e1-becc-4c78-af81-df5ac6e4da40",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 768
+        }
+      },
+      "source": [
+        "plt.figure(figsize: [12, 12])\n",
+        "var dis = 0\n",
+        "for image in 0...24 {\n",
```

Comment:
This is an index, not an image. Therefore something like `i` or `index` is better.

```suggestion
        "for i in 0...24 {\n",
```